modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -299,4 +245,102 @@ public static void setPosixFilePermissions(
 		}
 	}
 
+	private static Map<URL, String> _getResources(final String path)
+		throws IOException {
+
+		Map<URL, String> urlListToReturn = new HashMap<>();
+
+		Thread currentThread = Thread.currentThread();
+
+		final ClassLoader loader = currentThread.getContextClassLoader();
+
+		try (final InputStream inputStream = loader.getResourceAsStream(path);
+			final InputStreamReader inputStreamReader = new InputStreamReader(
+				inputStream, StandardCharsets.UTF_8);
+			final BufferedReader bufferedReader =
+				new BufferedReader(inputStreamReader)) {
+
+			List<URL> urls;
+
+			try (Stream<String> lines = bufferedReader.lines()) {
+				urls = lines.map(
```

Comment:
in this stream you create a list of urls, and then you check for null/empty and then you loop over those URLs again and check if they are not null and then do a if/else.  You shoudl continue this in the stream, since you are just streaming over the urls, which you could do a as a `filter`